packages/gatsby-source-filesystem/src/create-file-path.js

```diff
@@ -43,8 +43,8 @@ module.exports = ({
     slash(basePath),
     slash(fileNode.relativePath)
```

Comment:
Also, it looks like `absolutePath` would be more logical here, instead of `relativePath`.

Reply:
No we want relative as we're using it to generate the path so absolutePath would make paths that a) vary on where you're building the site and b) would be really long.